### ISSUE TITLE
Create Staging and Prod deployments for sign-in-service

### ISSUE THREAD

**Issue by vidjovanovic** [department-of-veterans-affairs/va.gov-team#65627]
Sign-in-service deployment for staging and prod must be created.

- [x] Ensure MFA is configured on account for tf apply
- [x] Ensure that staging tf components exist
- [ ] Ensure that prod tf components exit
- [x] Apply staging tf components
- [ ] Apply prod tf components
- [x] Update staging in VFS ArgoCD repo
- [ ] Update prod in VFS ArgoCD repo
- [x] Ensure previous components for sign-in-service that Tim created are removed

**Comment by vidjovanovic** [comment]
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2264
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/pull/2271

**Comment by vidjovanovic** [comment]
Ran apply via the makefile on sign-in-service staging components.

**Comment by vidjovanovic** [comment]
https://github.com/department-of-veterans-affairs/devops/pull/13593

Prod components for Sign-in-service

https://github.com/department-of-veterans-affairs/vsp-infra-argocd/pull/226

Staging environment in ArgoCD

**Comment by vidjovanovic** [comment]
still waiting on reply from vsp-operations, will submit another ticket.

**Comment by parkerbees** [comment]
Moved to Jira. Closing.
